cmd/podman/containers/create.go

```diff
@@ -237,17 +237,20 @@ func pullImage(imageName string) (string, error) {
 		imageMissing = !br.Value
 	}
 
-	if cliVals.Platform != "" {
-		if cliVals.OverrideArch != "" || cliVals.OverrideOS != "" {
-			return "", errors.Errorf("--platform option can not be specified with --override-arch or --override-os")
-		}
-		split := strings.SplitN(cliVals.Platform, "/", 2)
-		cliVals.OverrideOS = split[0]
-		if len(split) > 1 {
-			cliVals.OverrideArch = split[1]
+	if cliVals.Platform != "" || cliVals.Arch != "" || cliVals.OS != "" {
```

Comment:
@rhatdan can you add a `// FIXME` that states that all this logic should go into the backend? While this fix works for the clients, it does not work for the server.

I'd prefer something similar to https://github.com/vrothberg/libpod/commit/6aa9622350853aa34ec3423f0ae1dd1fd730532e but I never managed to get back to it.  I would love to remove the `pull` entirely (as the backend does that already) but that would lose the progress bars for the remote client.

Reply:
When we do the containers/common image, could we take care of this there?

Reply:
Yes we could do that. But that would be based on `libpod/image` which is currently not doing that it should. That's why I think that https://github.com/vrothberg/libpod/commit/6aa9622350853aa34ec3423f0ae1dd1fd730532e is closer to tackle the root of the problem.